packages/pkl.experimental.structuredRead/examples/https/https.pkl

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+/// Example using structureRead to get resources https
+/// resources. Not necessarily recommended usage.
+///
+/// Try `pkl eval fromHttps.pkl`
+module https
+
+import ".../structuredRead.pkl"
+
+Http {
+  `raw.githubusercontent.com/apple/pkl-pantry` {
```

Comment:
mmmm.... can I get a hash at the end of this. Having `pkl.experimental` tests break when tweaking the repo README seems... off.

Reply:
Whoops, I updated the other test with a hash, but forgot this one. I'll get that fixed